services/favourites.js

```diff
@@ -1,110 +1,57 @@
-        method: 'GET',
-        headers: {
-          Authorization: token,
-          'Upgrade-Insecure-Requests': 1
-        }
-      })
-        .then((response) => {
-          const { status, statusText } = response;
-          if (status === 200) return response.json();
-
-          const errorObject = {
-            errors: {
-              status,
-              details: statusText
-            }
-          };
-          throw errorObject;
-        })
-        .then(data => resolve(data))
-        .catch(errors => reject(errors));
-    });
-  }
-
-  /**
-   * Creates a new favourite item attached to the current user
-   * @param {*} token User's token
-   * @param {*} resourceId Id of the resource
-   * @param {*} resourceType Resource's type (can be dataset, layer or widget)
-   */
-  static createFavourite(token, resource = {}) {
-    const { resourceId, resourceType } = resource;
-
-    return new Promise((resolve, reject) => {
-      fetch(`${process.env.WRI_API_URL}/favourite`, {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-          Authorization: token
-        },
-        body: JSON.stringify({
-          application: process.env.APPLICATIONS,
-          env: process.env.API_ENV,
-          resourceId,
-          resourceType
-        })
-      })
-        .then((response) => {
-          const { status, statusText } = response;
-
-          if (status === 200) return resolve();
-
-          const errorObject = {
-            errors: {
-              status,
-              details: statusText
-            }
-          };
-          throw errorObject;
-        })
-        .then(data => resolve(data))
-        .catch((errors) => { reject(errors); });
-    });
-  }
-
-  /**
-   * Deletes an existing favourite item attached to the current user
-   * @param {*} token User's token
-   * @param {*} resourceId Id of the resource
-   */
-  static deleteFavourite(token, resourceId) {
-    return new Promise((resolve, reject) => {
-      fetch(`${process.env.WRI_API_URL}/favourite/${resourceId}`, {
-        method: 'DELETE',
-        headers: { Authorization: token }
-      })
-        .then((response) => {
-          const { status, statusText } = response;
-
-          if (status === 200) return resolve();
-
-          const errorObject = {
-            errors: {
-              status,
-              details: statusText
-            }
-          };
-          throw errorObject;
-        })
-        .then(data => resolve(data))
-        .catch((errors) => { reject(errors); });
+      resourceId,
+      resourceType
+    },
+    {
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: token
+      }
     });
-  }
-}
 
-export default FavouritesService;
+/**
+ * Deletes an existing favourite item attached to the current user
+ * @param {*} token User's token
+ * @param {*} resourceId Id of the resource
+ */
+export const deleteFavourite = (token, resourceId) =>
```

Comment:
Add error handling as we do in other services. All services must log and throw errors (to console)